src/components/toc-item/toc-item.tsx

```diff
@@ -85,16 +85,16 @@ export class TocItem {
       <Host {...(isParent ? { parent: isParent } : {})} class={`${!showItem ? 'excluded' : ''}`}>
         <div class="indented">
           {isParent ? (
-            <button class="indented__toggle-btn" type="button" onClick={this.toggle.bind(this)}>
+            <button class="indented__toggle-btn" type="button" onClick={this.toggle}>
```

Comment:
```suggestion
            <button class="indented__toggle-btn" type="button" onClick={() => this.toggle()}>
```

Reply:
I see that ionic uses your style. Let's choose one and stick with it, rather than mixing.

Reply:
done